src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -105,6 +114,20 @@ public override string SubProtocol
                 websocketControl.SupportedProtocols.Add(subProtocol);
             }
 
+            if (MessageWebSocketClientCertificateSupported)
+            {
+                // Use the first app-provided client cert that can be successfully converted into a WinRT client cert (if any).
+                foreach (X509Certificate dotNetClientCert in options.ClientCertificates)
+                {
+                    RTCertificate winRtClientCert = ConvertDotNetClientCertToWinRtClientCert(dotNetClientCert);
+                    if (winRtClientCert != null)
```

Comment:
>:Are we OK with adding a UWP-specific exception here?

No. We do not add new UWP-specific exceptions like that since it will break devs depending on NETStandard behaviors.